src/subsystem.js

```diff
@@ -27,7 +27,7 @@ export function mapSubsystem(alias, context) {
     return null;
   }
 
-  const { CORALOGIX_ALIAS_MAPPING: aliasMapping } = env;
+  const { CORALOGIX_ALIAS_MAPPING: aliasMapping = '{}' } = env;
```

Comment:
without setting a default mapping, every invocation reports an error and logs a stack trace, because it is unable to parse `undefined`.